### ISSUE TITLE
Vault API 404 when connecting the Agent with Local setup 

### ISSUE THREAD

**Issue by sidharthbh8** [kerberos-io/vault#21]
So, I'm trying to connect the agent with the vault from persistence. When verifying, the system throws the error `Something went wrong while verifying the persistence :cloud.VerifyPersistence(kerberosvault): Something went wrong while verifying storage credentials: 404 page not found on verification.`  Also unable to access API endpoints from browser, direct requests to `http://localhost:80/api` returns a 404. Swagger indicates a missing cookie token.

The documentation specifies the API URL for Kerberos Vault as `http://{your-vault-domain}/api`.
My setup uses localhost:80.

**Comment by sidharthbh8** [comment]
Yes I can reach and have logged into the `localhost:80`, and I have added the storage provider and account in the vault

**Comment by sidharthbh8** [comment]
Hi @cedricve when trying to make a API call from swagger its throwing a error in response
```json
{
  "code": 401,
  "message": "cookie token is empty"
}

**Comment by cedricve** [comment]
The 401 indicates its an unauthorized request. Did you provide the appropriate headers to authenticate?

**Comment by hiteshjangid** [comment]
any updates on this ?
